emu-charter.md

```diff
@@ -25,6 +25,9 @@ EDHOC (Ephemeral Diffie-Hellman Over COSE) is a very compact and lightweight aut
 
 While TLS-based EAP mechanisms provide strong channel protections, if the client does not authenticate and validate the server's credentials properly (possibly owing to a lack of provisioned information necessary to undertake that validation), an EAP mechanism running over TLS that relies on passwords is vulnerable to client credential theft, much the same as password authentication over plain TLS is. The FIDO Alliance and the W3C have developed a passwordless authentication scheme known as FIDO2, which combines elements of the W3C's WebAuthn and FIDO's CTAP standards. The group will devise an EAP method suitable for use with passwordless authentication schemes such as the CTAP2 version of FIDO2.
 
+While some EAP methods can provide some privacy there still can be a leakage of information as to which networks a particular user is accessing. Privacy pass protocols and tokens provide mechanisms to protect the users privacy in this situation.  The group will work on an EAP method that provides privacy by preventing a visited network or service from knowing the identity of a user, and for keeping the identity provider for that user from tracking what networks or services a specific user is accessing.
```

Comment:
> protect the **users** privacy

Should be `user's`? That is, the above is missing an apostrophe (possessive noun)?